system/jlib/jfcmp.hpp

```diff
@@ -43,9 +43,9 @@ class jlib_decl CFcmpCompressor : public CSimpleInterfaceOf<ICompressor>
     virtual void setinmax() = 0;
     virtual void flushcommitted() = 0;
 
-    void initCommon()
+    void initCommon(size32_t initialSize)
     {
-        blksz = inma.capacity();
+        blksz = initialSize;
```

Comment:
No, it's intentional not to reallocate.
It's fine for this [single] buffer to have the default size.